build.fsx

```diff
@@ -87,22 +87,16 @@ Target "CleanDocs" (fun _ ->
 // Build library & test project
 
 Target "Build" (fun _ ->
-    !! (solutionFile + "*.sln")
-    |> MSBuildRelease "" "Rebuild"
+    //!! (solutionFile + "*.sln")
```

Comment:
Nit: we can remove this commented line as well as the other at line 151

Reply:
Sure…